var/spack/repos/builtin/packages/apple-glu/package.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Reply:
indeed, those packages have been sitting in my private repo for a while :wink:
now fixed.